File: project_in_c/s21_is_even.c
#include "s21_decimal.h"

/* Проверяет, является ли число value четным.
Возвращает 1 (Да) и 0 (нет).
*/

int s21_is_even(s21_decimal value) {
  int res = 1;
  if (value.bits[0] & 1) res = 0;
  return res;
}

File: interactive_tutorial/assets/js/data/code/s21_is_even.js
export const s21_is_even_code = `#include "s21_decimal.h"

/* Проверяет, является ли число value четным.
Возвращает 0 (Да) и 1 (нет).
*/

int s21_is_even(s21_decimal value) {
  int res = 0;
  if (value.bits[0] & 1) res = 1;
  return res;
}
`;